emcc.py

```diff
@@ -3474,6 +3474,7 @@ def consume_arg_file():
             settings.SEPARATE_DWARF = requested_level.split('=')[1]
           else:
             settings.SEPARATE_DWARF = True
+          settings.GENERATE_DWARF = 1
```

Comment:
Without this, `SEPARATE_DWARF` is not recognized as generating DWARF and other related logic in this PR doesn't work